akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardCoordinator.scala

```diff
@@ -645,6 +645,7 @@ abstract class ShardCoordinator(typeName: String, settings: ClusterShardingSetti
         state = state.updated(evt)
         gracefulShutdownInProgress -= ref
         regionTerminationInProgress -= ref
+        aliveRegions -= ref
```

Comment:
This seems like something we had missed previously but maybe I am misunderstanding something, @patriknw ?